test/codegen/newman/fixtures/formdataFileCollection.json

```diff
@@ -1,44 +1,47 @@
 {
-	"info": {
-		"_postman_id": "7da8f1cd-aeb0-420f-a989-d47db46b44b0",
-		"name": "Multipart/form-data FILE",
-		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
-	},
-	"item": [
-		{
-			"name": "Single/multiple file upload via form-data",
-			"request": {
-				"method": "POST",
-				"header": [],
-				"body": {
-					"mode": "formdata",
-					"formdata": [
-						{
+  "info": {
+    "_postman_id": "7da8f1cd-aeb0-420f-a989-d47db46b44b0",
+    "name": "Multipart/form-data FILE",
+    "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+  },
+  "item": [
+    {
+      "name": "Single/multiple file upload via form-data",
+      "request": {
+        "method": "POST",
+        "header": [],
+        "body": {
+          "mode": "formdata",
+          "formdata": [
+            {
               "key": "single file",
               "value": "",
               "type": "file",
-              "src": "<file path>"
-						},
-						{
+              "src": "/mnt/c/Users/INVICTUS/Desktop/GSoC/postman-code-generators/dummyFile1.txt"
```

Comment:
this should be `<file path>` There is a script which takes care of adding the path from the CI here.